roles/common/tasks/main.yml

```diff
@@ -20,6 +20,37 @@
     retries: 5
   when: ansible_distribution == "Ubuntu"
 
+- name: Register with RHN
+  redhat_subscription:
+    state: present
+    username: "{{ redhat.user }}"
+    password: "{{ redhat.pass }}"
+  when: ansible_distribution == "RedHat"
+
+- name: Pause for registration
+  pause:
+    seconds: 90
+
+- name: Add OSP pool
+  command: subscription-manager attach --pool={{ item }}
+  when: ansible_distribution == "RedHat"
+  with_items:
+  - 8a85f98158880f8d01588bde0f7c3c43
+
+- name: add OS respositories
+  command: subscription-manager repos --enable={{ item }}
+  when: ansible_distribution == "RedHat"
+  with_items:
+  - rhel-7-server-extras-rpms
```

Comment:
need to make this variable
redhat.osp_repos and redhat.ceph_repos, again redhat.ceph_repos should be enabled only on osd node where we will also attach to ceph poolid.

Please discuss variable structure with @scoop206